airflow/auth/managers/utils/fab.py

```diff
@@ -36,6 +37,7 @@
     "GET": ACTION_CAN_READ,
     "PUT": ACTION_CAN_EDIT,
     "DELETE": ACTION_CAN_DELETE,
+    "PATCH": ACTION_CAN_PAUSE,
```

Comment:
@vincbeck 
Reason to introducing `PATCH` is because `PUT` is being mapped to ACTION_CAN_EDIT here.
If there is other suggestion please let me know.